zerver/migrations/0001_initial.py

```diff
@@ -47,8 +53,8 @@ class Migration(migrations.Migration):
 CREATE TRIGGER zerver_message_update_search_tsvector_async
   BEFORE INSERT OR UPDATE OF subject, rendered_content ON zerver_message
   FOR EACH ROW EXECUTE PROCEDURE append_to_fts_update_log();
-"""),
-            ]
+""")
+        zulip_postgres_migrations = [full_text_search_migration]
 
```

Comment:
I'd put the script with a path like `scripts/setup/enable-pgroonga`, and the documentation for the `USING_PGROONGA` option would suggest that users need to run that script in order to use it.

Does that seem reasonable?